integration/containerd/cri/integration-tests.sh

```diff
@@ -319,6 +342,135 @@ TestContainerMemoryUpdate() {
 	testContainerStop
 }
 
+TestContainerSwap() {
+	if [[ "${KATA_HYPERVISOR}" != "qemu" ]] || [[ "${ARCH}" != "x86_64" ]]; then
+		return
+	fi
+
+	local container_yaml=${REPORT_DIR}/container.yaml
+
+	info "Test container with guest swap"
+
+	create_containerd_config "${containerd_runtime_test}" 1
+	sudo sed -i -e 's/^#enable_guest_swap.*$/enable_guest_swap = true/g' "${kata_config}"
+
+	# Test without swap device
+	testContainerStart 0
+	swappiness=$(crictl exec $cid cat /sys/fs/cgroup/memory/memory.swappiness)
```

Comment:
Isn't this assuming a cgroups v1 system? What about Fedora (cgroups v2)? Or `/proc/sys/vm/swappiness`.

Reply:
Updated.